src/server/routes/search.js

```diff
@@ -57,6 +61,10 @@ router.get('/', (req, res, next) => {
 		.catch(next);
 });
 
+/**
+ * Returns autocomplete results for a given user query. Can be further
```

Comment:
Given that you end one sentence with a period here, I'd recommend using them consistently at the end of all sentences. Consistent and will probably read better in generated documentation.